app/model/Google/OAuthId.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Google;
+
+use InvalidArgumentException;
+use Ramsey\Uuid\Exception\InvalidUuidStringException;
+use Ramsey\Uuid\Uuid;
+use function sprintf;
+
+final class OAuthId
```

Comment:
Tohle by mělo pouze dědit z `AggregateId` místo reimplementace.

Reply:
měl jsem to asi puvodne na uuidV1, změneno